examples/__tests__/react-hooks.js

```diff
@@ -6,53 +6,76 @@
  */
```

Comment:
I'm guessing you had to skip the pre-commit hook or something. This should have auto-formatted when you commit it. Could you run `npx kcd-scripts format` please?

Reply:
I ran it and it added a linebreak in `ISSUE_TEMPLATE.md` and `mock.react-transition-group.js`. Should I also add it to the commit?
![captura de tela de 2019-02-10 12-48-41](https://user-images.githubusercontent.com/5133846/52535269-4d00f080-2d33-11e9-9ad8-e1a9d19d68ab.png)
![captura de tela de 2019-02-10 12-48-23](https://user-images.githubusercontent.com/5133846/52535270-4d00f080-2d33-11e9-9e2c-70aef1f4bf89.png)